samples/apps/copilot-chat-app/importdocument/README.md

```diff
@@ -7,12 +7,12 @@ One of the exciting features of the Copilot Chat App is its ability to store con
 to [memories](https://github.com/microsoft/semantic-kernel/blob/main/docs/EMBEDDINGS.md) and retrieve
 relevant information from memories to provide more meaningful answers to users through out the conversations.
 
-Memories can be generated from conversations as well as injected from external sources, such as text documents.
-Ingesting documents to the service will make the App even more intelligent by having prior knowledge of very
+Memories can be generated from conversations as well as imported from external sources, such as text documents.
+Importing documents to the service will make the App even more intelligent by having prior knowledge of very
```

Comment:
"Importing documents enables Copilot Chat to have up-to-date knowledge of specific contexts, such as enterprise and personal data."